zh_tw.lang

```diff
@@ -0,0 +1,209 @@
+gui.xaero_confirm =確認
+gui.xaero_cancel =取消
+gui.xaero_back =返回
+gui.xaero_delete =刪除
+gui.xaero_OK =確定
+gui.xaero_add_edit =添加/編輯
+gui.xaero_new_waypoint =新的路徑點
+gui.xaero_initial =初始化
+gui.xaero_edit_waypoint =編輯路徑點
+gui.xaero_armour_status_settings =盔甲狀態設置
+gui.xaero_choose_a_preset =預設方案
+gui.xaero_choose_an_interface =界面
+gui.xaero_armour_status =盔甲狀態
+gui.xaero_notifications =提醒
+gui.xaero_minimap =小地圖
+gui.xaero_miscellaneous =雜項
+gui.xaero_edit_mode =編輯模式
+gui.xaero_instructions =教程/幫助
+gui.xaero_not_ingame =你必須先進入遊戲開啟編輯模式
+gui.xaero_welcome_edit_mode =歡迎使用編輯模式
+gui.xaero_howto_select =左鍵點擊選擇界面
+gui.xaero_howto_drag =左鍵單擊並初始化界面
+gui.xaero_howto_deselect =快捷取消選擇界面
+gui.xaero_howto_center = C鍵固定界面居中
+gui.xaero_different_centered =某些界面有著不同的居中方式
+gui.xaero_howto_flip = F鍵插入界面
+gui.xaero_howto_settings = S鍵快速訪問選擇的界面關聯設置
+gui.xaero_howto_preset =點擊<預定方案>選擇一個預定方案
+gui.xaero_howto_save =點擊<確認>保存更改
+gui.xaero_howto_cancel =點擊<取消>放棄你的更改
+gui.xaero_minimap_settings =小地圖設置
+gui.xaero_miscellaneous_settings =雜項設置
+gui.xaero_notification_settings =提醒設置
+gui.xaero_better_pvp_settings =更好的PvP設置
+gui.xaero_server_disabled =服務器可能禁止了某些Mod特性
+gui.xaero_waypoints =路徑點
+gui.xaero_box_potion_effects =藥水效果界面
+gui.xaero_box_armour_status =裝備狀態界面
+gui.xaero_box_sprint =一鍵衝鋒
+gui.xaero_box_sneak =潛行開關
+gui.xaero_box_notifications =屏幕提醒，某些低溫/低血量/爆炸/弓箭。
+gui.xaero_box_xp =經驗顯示
+gui.xaero_box_quick_use =快速使用動作條物品
+gui.xaero_box_archery =弓箭數量顯示
+gui.xaero_box_minimap =小地圖顯示
+gui.xaero_centered =居中：
+gui.xaero_press_c = C鍵
+gui.xaero_flipped =翻轉
+gui.xaero_press_f = F鍵
+gui.xaero_preset_fancy =幻想
+gui.xaero_preset_classic =普通
```

Comment:
Translates back as "normal" to me, which isn't correct. The classic preset isn't "normal".